mods/cnc/chrome/connection.yaml

```
@@ -62,6 +62,7 @@ Container@CONNECTIONFAILED_PANEL:
 					Width: PARENT_RIGHT
 					Height: 25
 					Align: Center
+					Text: ServerError
```

Since that is a panel title
```suggestion
					Text: Server Error
```
? (This should probably be adjusted in the common file as well.)